scripts/test-scoreboard.sh

```
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Full test-suite scoreboard. Runs all tests vs current build/debug/bun-debug,
+# writes /tmp/SCOREBOARD.md. Idempotent; safe to re-run anytime.
+set -uo pipefail
+cd /root/bun-5
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)